config/locations.yml

```diff
@@ -1,148 +1,137 @@
+# from https://github.com/ualbertalib/discovery/blob/master/config/locations.yml and https://www.neoslibraries.ca/member-libraries
 aegeo:  
   name: "Alberta Geological Survey"
   locationurl: "http://ags.aer.ca/about-the-ags/contact-us.htm"
 aginternet:  
   name: "Alberta Government Library - Internet"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
 aglcap:  
-  name: "Alberta Government Library"
+  name: "Alberta Government Library - Capital Boulevard"
   locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
-ahsah:  
-  name: "Alberta Health Services - Alberta Hospital Edmonton"
-  locationurl: 
-ahsahe:  
+aglcom:
+  name: "Alberta Government Library - Commerce Place"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+aglgwl:
+  name: "Alberta Government Library - Great West Life"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+agllb:
+  name: "Alberta Government Library - Labour Building"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+aglmnc:
+  name: "Alberta Government Library - Monenco Bldg Calgary"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+aglncc:
+  name: "Alberta Government Library - Neil Crawford Centre"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+aglssp:
+  name: "Alberta Government Library - 107 Street"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+agltpn:
+  name: "Alberta Government Library - Telus Plaza North"
+  locationurl: "https://www.servicealberta.ca/alberta-government-library.cfm"
+ahsach:
+  name: "Alberta Health Services - Alberta Children's Hospital"
+  locationurl: "http://krs.libguides.com/aboutus/ach"
+ahsahe:
   name: "Alberta Health Services - Alberta Hospital Edmonton"
   locationurl: 
 ahscenten:  
   name: "Alberta Health Services - Centennial Centre"
   locationurl: 
-ahserc:  
-  name: "Alberta Health Services - Education Resource Centre"
```

Comment:
This was more specific than what was in Discovery [Alberta Health Services].  Which is correct?